android/src/main/java/org/mozilla/experiments/nimbus/FakeFeatures.kt

```diff
@@ -0,0 +1,154 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.experiments.nimbus
+
+import android.app.AlertDialog
+import android.app.Application
+import android.graphics.Color
+import android.widget.Button
+
+// This file shows some features being configured with variables coming from Nimbus.
+// This code is an example of what might be written by application developers.
+
+val context = Application()
+val nimbus = Nimbus()
+
+////////////////////////////////////////////////////////////////////////////////////////////////
+// Configuring a simple button.
+//
+// In reality we'll not be doing anything this complex with something so simple.
+////////////////////////////////////////////////////////////////////////////////////////////////
+
+fun configureButtonWithoutCodeGen() {
+    exampleJson = """
+        {
+            "buttonColor": "red",
+            "buttonText": "treatment_button_label"
+        }
+    """.trimIndent()
+
+    // config and lazyConfig are ostensibly the same shape, perform the JSON look up
+    // at different times.
+    val config = nimbus.getFeatureVariables("button-feature")
+
+    val button = Button(context).apply {
+        text = config.getText("buttonText") ?: "Ok then"
```

Comment:
is the idea here you would want to define inline defaults sometimes, like for very simple features?